assets/js/blocks/product-categories/block.js

```diff
@@ -98,30 +99,42 @@ class ProductCategoriesBlock extends Component {
 		const selectId = `prod-categories-${ instanceId }`;
 
 		return (
-			<div className={ classes }>
-				{ isDropdown ? (
-					<Fragment>
-						<div className="wc-block-product-categories__dropdown">
-							<label className="screen-reader-text" htmlFor={ selectId }>
-								{ __( 'Select a category', 'woo-gutenberg-products-block' ) }
-							</label>
-							<select id={ selectId } ref={ this.select }>
-								<option value="false" hidden>
-									{ __( 'Select a category', 'woo-gutenberg-products-block' ) }
-								</option>
-								{ this.renderOptions( categories ) }
-							</select>
-						</div>
-						<IconButton
-							icon="arrow-right-alt2"
-							label={ __( 'Go to category', 'woo-gutenberg-products-block' ) }
-							onClick={ this.onNavigate }
-						/>
-					</Fragment>
+			<Fragment>
+				{ ! isEmpty( categories ) ? (
```

Comment:
Nitpicking here, but `categories` will always be an array if I'm reading the code correctly, so I think checking `categories.length > 0` would work as well and we avoid one lodash import. :slightly_smiling_face: